src/character.cpp

```diff
@@ -5058,19 +5058,12 @@ int Character::get_stored_kcal() const
 
 static std::string exert_lvl_to_str( float level )
 {
-    if( level <= NO_EXERCISE ) {
-        return _( "NO_EXERCISE" );
-    } else if( level <= LIGHT_EXERCISE ) {
-        return _( "LIGHT_EXERCISE" );
-    } else if( level <= MODERATE_EXERCISE ) {
-        return _( "MODERATE_EXERCISE" );
-    } else if( level <= BRISK_EXERCISE ) {
-        return _( "BRISK_EXERCISE" );
-    } else if( level <= ACTIVE_EXERCISE ) {
-        return _( "ACTIVE_EXERCISE" );
-    } else {
-        return _( "EXTRA_EXERCISE" );
+    for( const std::pair<const std::string, float> &activity_level : activity_levels_map ) {
+        if( level <= activity_level.second ) {
+            return _( activity_level.first );
```

Comment:
`exert_lvl_to_str()` is for debug output so actually it doesn't need to be translated.
```suggestion
            return activity_level.first;
```